run.sh

```diff
@@ -74,7 +74,7 @@ ID_FS_LABEL=${ID_FS_LABEL:-$(blkid -o value -s LABEL)}
 ID_FS_UUID=${ID_FS_UUID:-$(blkid -o value -s UUID)}
 TEMPLATE="${ID_FS_LABEL:-nolabel}_${ID_FS_UUID:-nouuid}_XXX"
 
-DIRECTORY=$(mktemp -d "/output/$TEMPLATE")
+DIRECTORY=$(mktemp -d "/output/.makemkv/$TEMPLATE")
```

Comment:
`/output/.makemkv` won't always exist so this script will have to mkdir it before mktemp runs.

```
robpol86@bosco25:makemkv (WyseNynja-master) $ mktemp -d /tmp/dne/ff_XXX
mktemp: failed to create directory via template ‘/tmp/dne/ff_XXX’: No such file or directory
```